p2p_wallet/Services/FeatureFlags/Features.swift

```diff
@@ -7,7 +7,7 @@
 
 public extension Feature {
     static let coinGeckoPriceProvider = Feature(rawValue: "coinGeckoPriceProvider")
-    static let buyScenarioEnabled = Feature(rawValue: "keyapp_buy_scenario_enabled")
+//    static let buyScenarioEnabled = Feature(rawValue: "keyapp_buy_scenario_enabled")
```

Comment:
you can remove it now

Reply:
Yes. I will wait for confirmation from product team. Thanks!